pkg/motor/create_account.go

```diff
@@ -50,13 +46,14 @@ func (mtr *motorNodeImpl) CreateAccount(request mt.CreateAccountRequest) (mt.Cre
 	mtr.DIDDocument = doc
 
 	// create Vault shards to make sure this works before creating WhoIs
-	mtr.callback.OnMotorEvent(ct.MotorCallbackMessage_MTR_VAULT_CREATED, false)
+	mtr.callback.OnMotorEvent("Registering new DIDDocument for account", false)
 	vc := vault.New()
 	if _, err := createWhoIs(mtr); err != nil {
 		return mt.CreateAccountResponse{}, fmt.Errorf("create account: %s", err)
 	}
 
 	// ecnrypt dscShard with DSC
+	mtr.callback.OnMotorEvent("Encrypting shards for Vault", false)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant